notebooks/aggregated_crashes.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# <nbformat>3.0</nbformat>
+#
+# Please convert this Python file to an IPython/Jupyter notebook using the
+# Makefile:
+#
+#  cd path/to/crashathon/notebooks
+#  make
+#
+# A recent IPython with notebook support is required to run the Make task.
+#
+# The notebook will write the result to files named in the form:
+#
+#   crashes_<channel>.json
+#
+# Once configured as scheduled Spark jobs they can be downloaded from the
+# Spark job cluster:
+#
+#   https://analysis-output.telemetry.mozilla.org/<job-name>/data/crashes_<channel>.json
+#
+# <markdowncell>
+
+# Crashathon -- a Firefox crash collator
+
+# <codecell>
+
+import datetime
+import ujson as json
+from moztelemetry import get_pings
+
+# <markdowncell>
+
+## Some configuration variables
+
+# <codecell>
+
+#: Number of days to fetch crash pings for
+DAYS = 30
+
+#: Fraction of pings to fetch
+FRACTION = 1
+
+#: The build channels to fetch and store crash pings for
+CHANNELS = [
+    'nightly',
+    'aurora',
+    'beta',
+    'release',
+]
+
+#: The time is now
+NOW = datetime.datetime.now()
+
+
+def get_crash_pings(channel, start_date, end_date, fraction):
+    pings = get_pings(
+        sc,  # noqa
+        app='Firefox',
+        channel=channel,
+        submission_date=(start_date, end_date),
+        build_id=('20100101000000', '99999999999999'),
```

Comment:
Not sure, been following what https://github.com/mfinkle/user-data-analytics/blob/master/android-clients.ipynb does, python_moztelemetry doesn't say anything about that, so I wonder if that's an optimization or just an oversight :-/